exporter/src/main/as/flump/export/EditFormatsWindow.mxml

```diff
@@ -41,20 +41,6 @@
         <s:GridColumn dataField="scaleFactorsString" headerText="Scale Factors"/>
         <s:GridColumn dataField="textureBorder" headerText="Texture Border"/>
         <s:GridColumn dataField="maxAtlasSize" headerText="Max Atlas Size"/>
-        <s:GridColumn dataField="optimize" headerText="Optimize">
-          <s:itemEditor>
-            <fx:Component>
-              <s:ComboBoxGridItemEditor>
-                <s:dataProvider>
-                  <s:ArrayList>
-                    <fx:String>Memory</fx:String>
-                    <fx:String>Speed</fx:String>
-                  </s:ArrayList>
-                </s:dataProvider>
-              </s:ComboBoxGridItemEditor>
-            </fx:Component>
-          </s:itemEditor>
-        </s:GridColumn>
```

Comment:
Hmm I just checked some textures with PVRTexTool and you're right they do look ugly. Ill think about it a bit more and if I dont fiind a nice solution I'l add back the option to minimize the number of texture atlases